wodles/azure/azure_services/storage.py

```diff
@@ -133,7 +125,7 @@ def start_storage(args):
 
 def get_blobs(
     container_name: str,
-    blob_service: BlockBlobService,
+    service_client: BlobServiceClient,
```

Comment:
We should remove the `next_marker` parameter since it is no longer part of the `blobs` object.